frontend/plugins/vuex-orm-axios.js

```diff
@@ -70,13 +70,13 @@ export default ({ $axios, app }) => {
         break;
       case 404:
         Notification.dispatch("notify", {
-          message: "Warning: " + error.response.data.detail,
+          message: "Warning: " + messageDetail.params.detail,
           type: "warning",
         });
         break;
       default:
         Notification.dispatch("notify", {
-          message: error,
+          message: "Error: " + messageDetail.params.detail,
```

Comment:
We could apply your suggestion here, isn't it?
```suggestion
          message: `Error:  $messageDetail.params.detail`,
```

Reply:
yes, but don't forget the brackets  around the variable : `Error:  ${messageDetail.params.detail}`,